app/controllers/email_templates_controller.rb

```diff
@@ -14,8 +14,13 @@ def edit
   def update
     template = EmailTemplate.find(params[:id])
     template.update(template_params)
-    template.save!
-    redirect_to email_templates_path
+    if template.save
+      flash[:success] = "Updated #{template.title} template successfully."
+      redirect_to email_templates_path
+    else
+      flash[:alert] = "Failed to save #{template.title} template: " + template.errors.full_messages.join(", ")
```

Comment:
codecov: Added lines #L21 - L22 were not covered by tests

Reply:
good catch, I forgot to test the happy path. Added a test case to features/email_template.feature in the latest commit